source/images.d.ts

```diff
@@ -0,0 +1,13 @@
+/* TypeScript requires declaring files which are imported to
```

Comment:
Hrm, let's move this and `vue-shims.d.ts` to `tools/` somewhere. With the multi-design system approach, `source/` should have just clean design systems folders right below it. I'm open to other folders/setup for these kinds of files.

Reply:
Moved to tools/typings.